actions/push-to-dockerhub/action.yaml

```diff
@@ -0,0 +1,63 @@
+name: Publish Docker image
+description: Publish docker images to DockerHub
+
+inputs:
+  repository:
+    description: |
+      The caller workflow's repository
+  tags:
+    description: |
+      List of Docker images to be pushed.
+    required: true
+  build_path:
+    description: |
+      Path to Dockerfile, which is used to build the image.
+    default: "."
+
+runs:
+  using: composite
+  steps:
+    - name: Check out the repo
+      uses: actions/checkout@v4
+
+    - name: Get secrets for DockerHub login
+      id: get-secrets
+      uses: grafana/shared-workflows/actions/get-vault-secrets@main
+      with:
+        common_secrets: |
+          DOCKERHUB_USERNAME=dockerhub:username
+          DOCKERHUB_PASSWORD=dockerhub:password
+
+    - name: Log in to Docker Hub
+      uses: docker/login-action@f4ef78c080cd8ba55a85445d5b36e214a81df20a
+      with:
+        username: ${{ env.DOCKERHUB_USERNAME }}
+        password: ${{ env.DOCKERHUB_PASSWORD }}
+
+    - name: Extract metadata (tags, labels) for Docker
+      id: meta
+      uses: docker/metadata-action@9ec57ed1fcdbf14dcef7dfbe97b2010124a938b7
+      with:
+        images: ${{ inputs.repository }}
+
+    - name: Resolve tags
```

Comment:
Do you really need this step? Wouldn't it also work if you pass the tags int othe `docker/metadata-action` and use the output from there?

Reply:
Not sure, since the format they'll come from is:
```
tags: |-
"foo"
"bar"
```

I'll try though.

Reply:
Works indeed @zerok, thanks for the suggestion 🙇

Reply:
🥳 Used that a couple of days ago when rewriting the CI pipeline for one of my projects ^_^